src/frontend/screens/Settings/components/Tools/index.tsx

```diff
@@ -23,8 +23,7 @@ export default function Tools() {
     return <></>
   }
 
-  type Tool = 'winecfg' | string
```

Comment:
I wonder if it would be better to add the different strings we use when calling `callTools` instead of using just `string`? seems like a really limited number of possible values, sounds like an easy win for this type